ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandStoreTest.java

```diff
@@ -16,285 +16,110 @@
-        new ConsumerRecord<>("topic", 0, 0, terminated, new Command(
-            "terminate query 'queryId'", Collections.emptyMap(), Collections.emptyMap()
-    )))));
-
-    EasyMock.expect(commandConsumer.partitionsFor(COMMAND_TOPIC)).andReturn(Collections.emptyList());
-    EasyMock.expect(commandConsumer.poll(anyObject())).andReturn(records)
-        .andReturn(new ConsumerRecords<>(Collections.emptyMap()));
-    EasyMock.replay(commandConsumer);
-
-    final CommandStore commandStore = createCommandStore();
-    final RestoreCommands restoreCommands = commandStore.getRestoreCommands();
-    assertThat(restoreCommands.terminatedQueries(), equalTo(Collections.singletonMap(new QueryId("queryId"), terminated)));
-  }
-
-  private void setupConsumerToReturnCommand(final CommandId commandId, final Command command) {
-    reset(commandConsumer);
-    expect(commandConsumer.poll(anyObject(Duration.class))).andReturn(
-        new ConsumerRecords<>(
-            Collections.singletonMap(
-                new TopicPartition("command-topic", 0),
-                Collections.singletonList(
-                    new ConsumerRecord<>(
-                        "command-topic", 0, 0, commandId, command)
-                ))
-        )
-    ).times(1);
-    replay(commandConsumer);
-  }
-
-  private CommandStore createCommandStoreThatAssignsSameId(final CommandId commandId) {
-    final CommandIdAssigner commandIdAssigner = mock(CommandIdAssigner.class);
-    expect(commandIdAssigner.getCommandId(anyObject())).andStubAnswer(
-        () -> new CommandId(commandId.getType(), commandId.getEntity(), commandId.getAction())
-    );
-    replay(commandIdAssigner);
-    return createCommandStore(commandIdAssigner);
-  }
-
-  private CommandStore createCommandStore() {
-    return createCommandStore(new CommandIdAssigner(new MetaStoreImpl(new InternalFunctionRegistry())));
-  }
+  public void shouldGetRestoreCommands() {
+    // Given:
+    final RestoreCommands restoreCommands = Mockito.mock(RestoreCommands.class);
+    Mockito.when(commandTopic.getRestoreCommands(Mockito.any(Duration.class))).thenReturn(restoreCommands);
 
-  private CommandStore createCommandStore(final CommandIdAssigner commandIdAssigner) {
-    return new CommandStore(
-        COMMAND_TOPIC,
-        commandConsumer,
-        commandProducer,
-        commandIdAssigner);
-  }
+    // When:
+    final RestoreCommands restoreCommands1 = commandStore.getRestoreCommands();
 
-  private List<Pair<CommandId, Command>> getPriorCommands(final CommandStore command) {
-    final RestoreCommands priorCommands = command.getRestoreCommands();
-    final List<Pair<CommandId, Command>> commands = new ArrayList<>();
-    priorCommands.forEach(((id, cmd, terminatedQueries, droppedEntities) -> commands.add(new Pair<>(id, cmd))));
-    return commands;
+    // Then:
+    assertThat(restoreCommands1, equalTo(restoreCommands));
```

Comment:
Add another test method to verify that the correct duration is passed to the `commandTopic.getRestoreCommands()`